tests/integration/test_vulnerability_detector/conftest.py

```diff
@@ -2,6 +2,7 @@
 # Created by Wazuh, Inc. <info@wazuh.com>.
 # This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
 import pytest
+
```

Comment:
? Delete this new line

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/2847/commits/349274a11fb90a6a8421a443306f248ef179d68e